### DISCUSSION TITLE
Utilização do Popover

### DISCUSSION THREAD

**Discussion by Smmartinez** [po-ui/po-angular#1622] [Angular]
Bom dia! estou tentando utilizar o Popover mas ocorre o erro:ERROR TypeError: Cannot read properties of undefined (reading 'addEventListener')

Precisa adicionar alguma dependência ou modulo para executar? Eu utilizei o stackBlitz e não gera o mesmo erro que eu.

Obrigado

**Comment by rafaellmarques** [comment]
Bom dia @Smmartinez , tudo bem?
Você pode nos fornecer o link desse stackBlitz para darmos uma olhada?
